lms/static/js/toggle_login_modal.js

```diff
@@ -92,7 +92,11 @@
                         $notice.show().html(notice);
             // This is for activating leanModal links that were in the notice. We should have a cleaner way of
             // allowing all dynamically added leanmodal links to work.
-                        $notice.find('a[rel*=leanModal]').leanModal({top: 120, overlay: 1, closeButton: '.close-modal', position: 'absolute'});
+                        $notice.find('a[rel*=leanModal]').each(function() {
+                            var $link = $(this),
+                                closeButton = $link.data('modalCloseButtonSelector') || '.close-modal';
+                            leanModal({top: 120, overlay: 1, closeButton: closeButton, position: 'absolute'});
+                        });
```

Comment:
This change was made for consistency with another change I pushed out last week that made it possible to specify the closeButton selector via a data attribute.